packages/web/src/components/spa/JobsSection/jobs_section.css

```diff
@@ -1,5 +1,436 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+:root {
+    --main-color: #32baf4;
+    --shadow-color: #467edd;
+    --background-color: #f7f7f7;
+    --font-family: 'Poppins';
+    --text-color: #333;
+    --placeholder-color: white;
+}
+
 .jobs-container {
-    height: 100vh;
+    height: 125vh;
     width: 100vw;
     background: linear-gradient(rgb(21, 76, 121), rgb(118, 181, 197));
 }
+
+/*
+    Code for a search bar. May be implemented in the future. 
+*/
+
+/* .searchBar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-top: 2%;
+    padding-left: 11%;
+    padding-right: 11%;
+    align-items: center;
+}
+
+.box {
+    height: 6vh;
+    display: flex;
+    cursor: pointer;
+    padding: 1% 1.5%;
+    border-radius: 100%;
+    align-items: center;
+    box-shadow: 0em 1em 2.5em rgba(0,0,0,0.3);
+    background-image: linear-gradient(
+        to right,
+        #32baf4,
+        #467edd,
+        #3f86ed,
+        #47acdf,
+        #24cc8e
+    );
+}
+
+.box:hover input {
+    width: 25vw;
+}
+
+.box input {
+    color: white;
+    width: 0;
+    outline: none;
+    border: none;
+    font-weight: 500;
+    transition: 0.8s;
+    background: transparent;
+    font-family: var(--font-family);
+}
+
+::placeholder{
+    color: var(--placeholder-color);
+} */
+
+/*
+    Code for a search bar. May be implemented in the future. ^^
+*/
+
+.jobs {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding-top: 2%;
+    justify-content: space-evenly;
+    margin-left: 6%;
+    margin-right: 8%;
+}
+
+.job-box {
+    height: 51vh;
+    width: 21vw;
+    box-shadow: 0% 1% 2.5% var(--shadow-color);
+    border-radius: 11.5%;
+    transition: all 0.3s;
+    position: relative;
+    background-color: var(--background-color);
+}
+
+.job-box:hover {
+    transform: translateY(-3%) rotate(3deg);
+    box-shadow: 0 1.5% 3% #467edd;
+    cursor: pointer;
+    z-index: 1;
+}
+
+.job-box::before {
+    content: '';
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    border-radius: 10%;
+    background-color: rgba(255, 255, 255, 0.1);
+    z-index: -1;
+    transform: scale(1.05);
+    transition: all 0.3s;
+}
+
+.job-box:hover::before {
+    transform: scale(1.1);
+}
+
+.photo img {
```

Comment:
Please, don't use such global class names. Call this one `jobs-photo-container` or reference it through it's parent div.